packages/dropdowns/src/elements/combobox/Combobox.tsx

```diff
@@ -177,15 +177,6 @@ export const Combobox = forwardRef<HTMLDivElement, IComboboxProps>(
       'Options'
     );
     const triggerProps = {
-      isAutocomplete,
-      isBare,
-      isCompact,
-      isEditable,
-      isLabelHovered,
-      isMultiselectable,
-      maxHeight,
-      focusInset,
-      validation,
       ...(getTriggerProps({
```

Comment:
```suggestion
      const triggerProps = getTriggerProps({
```

I don't think the spread is needed any longer